src/EA.Weee.Domain/AatfReturn/Return.cs

```diff
@@ -25,7 +25,7 @@ public Return(Operator aatfOperator, Quarter quarter, string createdBy)
             CreatedDate = SystemTime.UtcNow;
         }
 
-        public void UpdateSubmitted(string submittedBy)
+        public virtual void UpdateSubmitted(string submittedBy)
```

Comment:
Just a curious question. If this is not being over ridden anywhere, why is it virtual?